Provisioning/Vault/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+kubecontext = "kind-kind"
```

Comment:
This file should not even exist. Have a look at https://github.com/adfinis/Vault-Playground-on-Kubernetes/pull/7

Reply:
replaced it with an example tfvars file